Generation of bgp_configs.go fails with python 3.6

I'm trying to generate bgp_configs.go on ubuntu 18.04, but it's failing. There is an issue with bgpyang2golang.py that I have fixed locally (see patch below), but the generated bgp_configs.go still fails with the following errors when building gobgp(d).

```
# github.com/osrg/gobgp/internal/pkg/config
../../../go/src/github.com/osrg/gobgp/internal/pkg/config/bgp_configs.go:6111:27: l.Config undefined (type Neighbor has no field or method Config)
../../../go/src/github.com/osrg/gobgp/internal/pkg/config/bgp_configs.go:6114:38: r.Config undefined (type Neighbor has no field or method Config)
../../../go/src/github.com/osrg/gobgp/internal/pkg/config/default.go:53:6: n.State undefined (type *Neighbor has no field or method State)
../../../go/src/github.com/osrg/gobgp/internal/pkg/config/default.go:78:6: n.Config undefined (type *Neighbor has no field or method Config)
../../../go/src/github.com/osrg/gobgp/internal/pkg/config/default.go:79:4: n.Config undefined (type *Neighbor has no field or method Config)
../../../go/src/github.com/osrg/gobgp/internal/pkg/config/default.go:81:5: n.Config undefined (type *Neighbor has no field or method Config)
../../../go/src/github.com/osrg/gobgp/internal/pkg/config/default.go:83:5: n.Config undefined (type *Neighbor has no field or method Config)
../../../go/src/github.com/osrg/gobgp/internal/pkg/config/default.go:86:3: n.State undefined (type *Neighbor has no field or method State)
../../../go/src/github.com/osrg/gobgp/internal/pkg/config/default.go:86:21: n.Config undefined (type *Neighbor has no field or method Config)
../../../go/src/github.com/osrg/gobgp/internal/pkg/config/default.go:88:6: n.Config undefined (type *Neighbor has no field or method Config)
../../../go/src/github.com/osrg/gobgp/internal/pkg/config/default.go:88:6: too many errors

```

```
diff --git a/tools/pyang_plugins/bgpyang2golang.py b/tools/pyang_plugins/bgpyang2golang.py
index 96e24d87..2fbf12f9 100644
--- a/tools/pyang_plugins/bgpyang2golang.py
+++ b/tools/pyang_plugins/bgpyang2golang.py
@@ -766,7 +766,7 @@ def generate_header(ctx, fd):
     print('import (', file=fd)
     print('"fmt"', file=fd)
     print('', file=fd)
-    print('"github.com/osrg/gobgp/packet/bgp"', file=fd)
+    print('"github.com/osrg/gobgp/pkg/packet/bgp"', file=fd)
     print(')', file=fd)
     print('', file=fd)

```

--- a/tools/pyang_plugins/README.rst
+++ b/tools/pyang_plugins/README.rst
@@ -5,7 +5,7 @@ openconfig yang files (see https://github.com/openconfig/public).
 
 Prerequisites
 =============
-Please confirm $GOPATH is configured before the following steps.
+Please confirm $GOPATH is configured before the following steps. Note that you need to use Python 2.
 
 How to use
 ==========

